packages/ember/addon/index.ts

```diff
@@ -58,4 +92,5 @@ function createEmberEventProcessor(): void {
   }
 }
 
+export default Sentry;
```

Comment:
We don't use default exports in any of the SDKs. Is there any specific reason to do that in ember integration? (i have close to 0 experience with this framework).
If not, then I think that consistency trumps anything else in this case and we should stick with the same usage everywhere as much as we can (eg. you are not able to do
```js
import Sentry from '@sentry/browser';
```
in the same way as you can do
```js
import Sentry from '@sentry/ember';
```
right now)

Reply:
No, it shouldn't be required, I'm in favor of consistency where possible too 👍, will fix.